### PR TITLE
Publish A-102375

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes DOC-7607

### CODE REVIEW

File: advisories/a102375.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING sql.query_cache.enabled = false;
+~~~
+
+However, note that this may degrade the performance of the cluster.
+
+## Impact
+
+Some customers running identical queries with different roles to access tables with the same name in different databases could experience spurious [privilege](../v22.2/security-reference/authorization.html#privileges) errors on CockroachDB v22.1.19 and v22.2.b8 with the query cache enabled.
```
Comment (rytaft) [inline advisories/a102375.md:47]: v22.2.b8 -> v22.2.8

**Review by rytaft — APPROVED** [review]
LGTM, just one small typo below. Thanks!

**Review by rytaft — APPROVED** [review]
LGTM, just one small typo below. Thanks!

**Comment by rmloveland** [comment]
thanks @rytaft !

I know you just reviewed this but for the sake of explicitness: I am passing along to my docs review partner now and targeting publish tomorrow as stated in the document.  Based on your review and the Slack convo re: timing it seems like that is OK.  Please shout if not tho

File: advisories/a102375.md (hunk 1 above)
Comment (rmloveland) [reply advisories/a102375.md]: fixed, thanks!

**Comment by rytaft** [comment]
Yep, sounds good to me. Thanks!

File: advisories/a102375.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- `SELECT * FROM database_name.schema_name.table_name`
+
+Another option, if an upgrade is not possible, is to disable the query cache with the following command:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING sql.query_cache.enabled = false;
+~~~
+
+However, note that this may degrade the performance of the cluster.
```
Comment (ianjevans) [inline advisories/a102375.md:43]:
```suggestion
Disabling the query cache may degrade the performance of the cluster, however.
```

**Review by ianjevans — APPROVED** [review]
Looks good, just a phrasing nit suggestion.

**Pull request merged**